torch/testing/_internal/inductor_utils.py

```diff
@@ -74,6 +74,7 @@ def _check_has_dynamic_shape(
 def skipDeviceIf(cond, msg, *, device):
     if cond:
         def decorate_fn(fn):
+            @functools.wraps(fn)
```

Comment:
this is needed in order for the special `_expected_failure` attributes to be propagated up